app/shared/components/title-with-quote/TitleWithQuote.tsx

```diff
@@ -23,11 +24,12 @@ const TitleWithQuote = ({
   color,
   quoteBlockSx,
   quoteSectionSx,
-  quoteWidth
+  quoteWidth,
+  dataTestId
 }: TitleWithQuoteProps) => {
   const styles = getStyles(color);
   return (
-    <Box sx={styles.mainContainer}>
+    <Box sx={styles.mainContainer} data-testid={dataTestId}>
       <Box sx={styles.titleSection}>
         <Typography variant="h2" sx={styles.titleText}>
```

Comment:
You probably need to add concatenation next:
```suggestion
        <Typography variant="h2" sx={styles.titleText} data-testid={`${dataTestId}-title`}>
```
Fixes for other places too.

Reply:
I’ve added data-testid attributes to all child components following the naming convention. Please review it again to make sure everything looks good.